sdk/identity/ci.yml

```diff
@@ -20,7 +20,4 @@ pr:
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    ServiceDirectory: identity
-    Artifacts:
```

Comment:
Does this mean we don't plan to produce any release artifacts any longer?

Reply:
That's correct, we don't actually produce any build artifacts because SwiftPM dependencies are consumed as source code and acquired directly from the Git repo that contains it.

The release artifact stuff was already removed from the template in #354, we just missed removing the variables from the service-level CI configs.

Reply:
We'll probably end up borrowing a bit from the C/C++ repos around how they tag their repository. I was looking through @danieljurek's PR around that this morning. The ```Package.swift``` file doesn't actually contain a version number, it is all implied from the ref containing the file. So I think we'll probably end up with a model where we have a text file in the repository which specifies the next version number and when we publish we simply read that into a variable and tag the repository.

We could consider publishing XCFrameworks as well, but since they are an XCode 11 construct, I think most people would use Swift Package Manager since its easier to manage.

Reply:
... still need the approval gates and all that jazz.

Reply:
Here's more context... the use case for iOS may be different: 

We ended up dropping `version.txt` in favor of writing the version information into constants in a header file. While the `version.txt` approach was rather simple from an EngSys perspective (it's easier to read text from a file in CMake and publishing pipeline code), @JeffreyRichter rightly pointed out that we shouldn't tie customers to our build system in C. This is because customers can use a variety of tools to build our C SDKs into their projects and some of those tools do not support CMake. 

The current approach in C uses a `version.h` file and we make the version information available to the build system and publishing pipelines using a regex to select the interesting parts of the text in the `version.h` file. We also use regexes to increment the version after shipping. Hardcoding the version in this way means that version information is always available to the core library for sending version information in the user agent string, among other scenarios. 

We'll eventually take the same approach with C++ pending SDK engineer work in C++ to properly hardcode the version.

Reply:
That makes sense to me too. Thanks. I do wonder the level of confusion that could occur if we ever decided to "reversion" a commit which is totally possible with Swift Package Manager. The code could be telling itself it is 1.0.0, but the tag on the repo is saying 1.0.0-beta.1. Could make diagnosing issues interesting.

Hopefully we would never be so foolish :)